src/game.js

```diff
@@ -209,6 +218,14 @@
             // reset the frame counter
             frameCounter = 0;
             frameRate = ~~(0.5 + 60 / me.sys.fps);
+            
+            // set step size based on the updatesPerSecond
+            stepSize = (1000 / Math.min(me.sys.updatesPerSecond, me.sys.fps));
```

Comment:
From what I'm aware, `me.sys.fps` is the *target* fps, not actual fps. So updatesPerSecond will never run faster than the target display fps, which is the intention.

From my tests running a game at 20/60 fps, this code is working. The character moves at the same speed when the game runs at 20/60 fps as when it does when the game runs at 60/60 fps.